en/developer-docs/docs/develop-components/connect-to-remote-dependencies-while-developing.md

```diff
@@ -0,0 +1,141 @@
+To streamline local development and verification of Choreo components that depend on [Choreo connections](../../choreo-concepts/connections/), you can bridge your local environment with the remote/deployed project. This is facilitated by the Choreo CLI and VS Code extension, which create a secure connected environment enabling real-time testing and validation.
```

Comment:
Don't we have a heading for this page?

Reply:
The menu item heading seems to be automatically rendered as the page heading